### DISCUSSION TITLE
Get DNS from the iptv Vlan

### DISCUSSION THREAD

**Discussion by apedroheringer** [fabianishere/udm-iptv#45] [Q&A]
Hi, I use the same lan on my network for iptv and my computers, mobiles...
This lan uses my pihole as dns
Can I make the STB/Decoders recieve the DNS settings from the DHCP from my provider, instead from the DHCP from my lan (that server pihole address)
I imagine that if I create a separate vlan it would be easy to do this, but  I would like to use the same one...
Thank you!

**Comment by fabianishere** [comment]
This project cannot address your issue unfortunately.

You might find luck in using [a custom dnsmasq config](https://www.reddit.com/r/Ubiquiti/comments/i9ft5u/so_heres_how_i_got_local_dns_records_working_on/) to send different DNS servers to your STB. However, at that point, why not just create a new VLAN for your STBs.

**Reply by apedroheringer** [reply]
I will just create a vlan. Thank you
